Assets/Scripts/Health.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+namespace FunMath
+{
+    public class Health : MonoBehaviour
+    {
+        [SerializeField]
+        [Range(0, 100)]
+        int health = 10;
+
+        public void TakeDamage(int damage)
```

Comment:
Probably rename it to modify health instead, to shelter against postive/negative number bugs